apps/web/utils/actions-client.ts

```diff
@@ -37,3 +39,22 @@ export async function onTrashThread(threadId: string) {
     });
   }
 }
+
+export async function onMarkUnmarkReadAction(
```

Comment:
The naming is a bit all over the place here. `onMarkUnmarkReadAction`

It's either mark and unmark. But simpler might just be: `onMarkReadAction`